### DISCUSSION TITLE
Modify Player Character

### DISCUSSION THREAD

**Discussion by NatKromer** [haven1433/HexManiacAdvance#204] [Q&A] [answered]
So, while I've figured out how to modify a lot of things with HexManiacAdvance, changing the player character themself seems to be something of a blind spot for the program, or at least for the tutorials and guides I've been able to find. 
How would one, for instance, go about changing which sprite(s) the player character uses (not just editing the sprites themselves, but changing them like you would for an NPC)? Or working with the player's trainer data? Or changing where the player spawns in at the start of the game? 
Any advice is appreciated.

**Comment by haven1433** [comment, marked as answer]
I'm not sure I understand what you're asking. HMA let's you edit the data already in the game, but has limited capacity to add features that aren't already in the game.

There's no part in the vanilla games that change the character trainer backsprite mid game, or change the character overworld sprite mid game. Unless I'm mistaken, in which case I suggest you examine how it's done in the game, and then you can copy the logic.

**Reply by NatKromer** [reply]
I didn't mean changing it mid-game. I just meant setting which sprite the player character uses, like you can for enemy trainers or events in the overworld. Like, is that a thing?
EDIT: Never mind, I think I see why that wouldn't work. Apologies for wasting your time.
